docs/src/building.md

```diff
@@ -12,8 +12,8 @@
 
 ## Build steps
 
-* Git clone the repo (make sure to use `--recursive` option to fetch submodules):  
```

Comment:
Double spaces at the end here are intentional, that's how manual line breaks are done in markdown